src/components/DefineCompany/YourCompany.vue

```diff
@@ -15,10 +15,10 @@
                 v-if="companyNameChanges"
                 class="ma-2 pa-2"
                 color="primary"
```

Comment:
Color should be "#1669BB". Watch for the merge with Lekshmi's PR so you don't undo her change.

Reply:
kk, will do!